pkg/tsdb/cloudwatch/services/log_groups.go

```diff
@@ -36,7 +38,7 @@ func (s *LogGroupsService) GetLogGroups(req resources.LogGroupsRequest) ([]resou
 	result := []resources.ResourceResponse[resources.LogGroup]{}
 
 	for {
-		response, err := s.logGroupsAPI.DescribeLogGroups(input)
+		response, err := s.logGroupsAPI.DescribeLogGroupsWithContext(context.Background(), input)
```

Comment:
Rather than using a `context.Background()` here, we should get it passed into `GetLogGroups`.

Reply:
Thanks @iwysiu for the review. Will update it.

Reply:
Updated.